### DISCUSSION TITLE
Clarification on 'forge test' Command Usage

### DISCUSSION THREAD

**Discussion by DonkoTonev** [Cyfrin/foundry-full-course-cu#1146] [Q&A]
Hello everyone,

I am encountering an issue while trying to run the 'forge test' command with the '-m' flag. The error message suggests that '-m' is an unexpected argument. However, based on my understanding and the documentation, it seems like it should be a valid option.

Here is the command I am trying to run:

`forge test -m testFundFailsWithoutEnoughETH
`

Am I using the command incorrectly, or is there an issue with the current version of Forge? Any insights or suggestions would be greatly appreciated.

Thank you!

**Comment by 0xusmanf** [comment]
`-m` command has been updated to `--mt`, that's why its not working for you. Use `--mt`. Also keep an eye on [this file](https://github.com/Cyfrin/foundry-full-course-f23/blob/main/chronological-updates.md), it has all the updates you should know through out the course.
